src/harryPotterReader/Word.java

```diff
@@ -0,0 +1,23 @@
+package harryPotterReader;
+
+public class Word {
+	
+	private String word;
+	private int count = 1;
```

Comment:
put the `count = 1` in the constructor. This works but people don't like it as sometimes it messes stuff up. I won't go into details bc it is complicated to explain. (YOU WOULD GET POINTS OFF BC OF SOMETHING LIKE THIS ON THE AP TEST)